bangazonapi/views/profile.py

```diff
@@ -71,7 +71,7 @@ def list(self, request):
     def cart(self, request):
         """Shopping cart manipulation"""
 
-        current_user = Customer.objects.get(user=request.auth.user)
+        current_user = Customer.objects.get(user__id=request.auth.user.id)
```

Comment:
This line doesn't need to change, it was already working as expected

Reply:
Got confused when i saw it at a glance.  Change revised.